src/Php84/bootstrap.php

```diff
@@ -15,6 +15,14 @@
     return;
 }
 
+if (defined('CURL_VERSION_HTTP3') || PHP_VERSION_ID < 80200 && function_exists('curl_version') && curl_version()['version'] >= 0x074200) { // libcurl >= 7.66.0
```

Comment:
Why PHP 8.2.0 as boundary ? the PR you linked was merged in 8.4-dev so we might need the polyfill on 8.2 or 8.3

Reply:
@stof: If the `CURL_VERSION_HTTP3` constant does not exist although we're on PHP 8.2, we know for sure that we either don't have ext-curl or the linked libcurl is too old, so we don't need to call `curl_version()` anymore.

Reply:
https://github.com/php/php-src/pull/8720/files#diff-29336a112e08c5b5caa7950afb6601010867f40a4643a33263646504277f7b33R1195

this adds the constant if php is compiled with a recent enough version of curl